### ISSUE TITLE
Named graphs are global

### ISSUE THREAD

**Issue by sleepycat** [arangodb/arangodb#1724]
Lets say I have two databases "foo" and "foo_test", each containing collections "vertices" and "edges".

If I define a graph named "foo_graph" in the "foo" database I can see the graph populated with the data in the collections from the "foo" database and all is goodness and light.

Lets say I switch to my "foo_test" database where my vertices and edges collections are empty, and define a graph called "foo_graph".

When I click the newly created "foo_graph" in my "foo_test" database what is displayed is the "foo_graph" with the data from the "foo" database. 

Has Arango always worked this way (and I just never noticed?) or is this a bug?
Either way, this behaviour is very surprising.

**Comment by pluma** [comment]
Can you confirm that this is not just a bug in the web interface?

**Comment by sleepycat** [comment]
Yeah, with a little further tinkering it seems that is the case.
Graph operations in the empty test db return nothing, and the same query returns vertices in my dev db.

**Comment by dothebart** [comment]
could this be a duplicate of #1715 ?

**Comment by hkernbach** [comment]
@dothebart yes, i think so.

**Comment by pluma** [comment]
Looks like it. The problem seemed familiar I just wasn't sure what it was a duplicate of.

@sleepycat can you check whether the problem persists with the devel version?

**Comment by sleepycat** [comment]
I tried to compile devel to test it but it blows up:

```bash
--------------------------------------------------------------------------------
BUILDING V8
--------------------------------------------------------------------------------

cd ./3rdParty/V8-4.3.61 \
	&& export GYPFLAGS="-Dstandalone_static_library=1 -Dlinux_use_bundled_gold=0 -Dlinux_use_gold_flags=0" \
	&& make \
	CC="gcc" \
	CC.host="gcc" \
	CXX="g++" \
	CXX.host="g++" \
	LINK="g++" \
	CFLAGS="-O3 -fomit-frame-pointer -g  " \
	CXXFLAGS="-O3 -fomit-frame-pointer -g  " \
	library=static strictaliasing=off snapshot=off werror=no x64.release
make[1]: Entering directory '/home/mike/projects/arangodb_pkg-git/src/arangodb/3rdParty/V8-4.3.61'
make[1]: *** No rule to make target 'x64.release'.  Stop.
make[1]: Leaving directory '/home/mike/projects/arangodb_pkg-git/src/arangodb/3rdParty/V8-4.3.61'
Makefile:10340: recipe for target '.v8-build-64' failed
make: *** [.v8-build-64] Error 2
```

**Comment by kvahed** [comment]
Hi Mike,

Do you have some details for us? Could you post us config.log? Did you just clone/pull from the repository?

Cheers,
Kaveh

**Comment by dothebart** [comment]
It was an error in the V8 integration during the cmakification. We're trying to get rid of the old style configure, however, I've commited a fix for this.

**Comment by hkernbach** [comment]
Can we close this? The issue is a duplicate and is fixed in devel: https://github.com/arangodb/arangodb/issues/1715

**Comment by sleepycat** [comment]
I'm still seeing this issue in 2.8.6.

**Comment by pluma** [comment]
@hkernbach did you backport the fix to the 2.8 branch?

**Comment by hkernbach** [comment]
@pluma Nope, sorry. Now it is. So the fix will be available in a possible 2.8.7 release and 3.X.X.
